src/collections/blog/2022/2022-20-06-managing-multiple-clusters-with-meshery/index.mdx

```diff
@@ -0,0 +1,99 @@
+
+<p align="center">
+  <img src={Cluster2} alt="visualizer showing data of context1" class="slides-right" />
+</p>
+<p align="center">
+  <img src={Cluster1} alt="visualizer showing data of context2" class="slides-left" />
+</p>
+
+
+### Managing Meshery on multiple clusters
+
+Users can perform cluster related operations from the settings page like adding more clusters, removing data from existing clusters and removing existing clusters.
+
+<img src={Settings} alt="Settings page" class="slides-right" />
+
+<Link to="/meshery">Meshery</Link> also deploys its operator across the cluster it’s about to manage. This operator manages the lifecycle of a NATS server and MeshSync. MeshSync pumps the blood into Meshery’s core, in other words, it is responsible for watching all different types of resources by establishing a watch stream over each of them. MeshSync then pumps that data into the NATS server, of which Meshery server itself is a client. From there, Meshery server gets all the relevant data related to activities in the cluster.
+
+By default <Link to="/meshery">Meshery</Link> wants to be greedy and as much aware about your infrastructure as possible so it deploys its operator across each detected cluster. But you can fine tune this configuration by going over each one of them from the table as shown.
+
+<img src={Cluster} alt="cluster management" class="image-center-shadow" />
+
+If you disconnect your cluster and do not want to persist the data from that cluster then you can perform a fine grain deletion by deleting all MeshSync data(which are the kubernetes objects) for that specific cluster.
+ 
+<img src={Flush} alt="flushing meshsync data"  class="image-center-shadow" />
+
+
+
+
+
+## Future of multi cluster
+
+<Link to="/meshery">Meshery</Link> as an extension point to your infrastructure provides out of the box value by adding components which can be kubernetes specific, service mesh specific or custom components to add new functionality. We can now add multi-cluster specific components to provide more abstraction. This model can be used along with Meshery’s multi-mesh capabilities to give an overall multi-mesh multi-cluster experience to the user. For instance, your Istio service mesh spanning across multiple clusters can be abstracted and managed by Meshery using custom components such as VirtualGateway and VirtualDestinationRules. In this case, Meshery’s Istio adapter will handle the logic of converting a VirtualGateway into gateways across the clusters. This abstraction provides high value by powering the service mesh to span across the clusters while the Ops team can configure the mesh with minimal effort.
+
+
+Just like the above example, many such extension points are in Meshery to add logic into and add useful functionality. And as these extension points actually extend, Meshery will continue to give more and more power to your cloud native infrastructure.
```

Comment:
It's not "the above example", but "the example above".

Better yet, don't make these kinds of references, instead add a caption to the image and refer to it by caption name / figure number.